src/lib/node_status_service/node_status_service.ml

```diff
@@ -2,17 +2,6 @@ open Async
 open Core
 open Pipe_lib
 
-type catchup_job_states =
```

Comment:
I think it makes sense for this still to live here, why don't you make it an alias
```ocaml
type catchup_job_states = Transition_frontier.Full_catchup_tree.job_states =
  { ... }
```
Importantly, this ensures that we will have to update the code here if the contents of this record changes, and thus we will be immediately aware that the node status service should be able to handle it.